docs/contributor/testing.md

```diff
@@ -0,0 +1,31 @@
+## Running unit tests using CLI
```

Comment:
Provide just one testing.md in the root /docs directory.

Reply:
Done